buildkite/common

```diff
@@ -32,4 +32,5 @@ export APP_DOCKER_COMMIT_TAG=$ORG/hint:$GIT_ID
 export APP_DOCKER_BRANCH_TAG=$ORG/hint:$GIT_BRANCH
 export USER_CLI_COMMIT_TAG=$ORG/hint-user-cli:$GIT_ID
 export USER_CLI_BRANCH_TAG=$ORG/hint-user-cli:$GIT_BRANCH
-
+export USER_CLI_COMMIT_TAG_NEW=$ORG/hint-user-cli:$GIT_ID
```

Comment:
similar to below comment, this is the same as `USER_CLI_COMMIT_TAG` so can be removed and the latter used above on line [27](https://github.com/mrc-ide/hint/pull/727/files#diff-42a67217e1a50efab8f2005c8a82c1b1e9158eb23a8baee4ddfe6542271d1233R27)